src/foam/nanos/bench/benchmarks/scripts.jrl

```diff
@@ -30,6 +30,122 @@ runner.execute(x);""",
 	"lastModifiedBy": 1348
 })
 
+//MDAO update method 
+
+p({
+  "class": "foam.nanos.script.Script",
+  "id": "benchmarkMDAOPut",
+  "description": "Benchmark MDAO put",
+  "code": """import foam.nanos.bench.*;
+    import foam.nanos.bench.benchmarks.*;
+    import net.nanopay.bench.*;
+
+    x = x.put("pmLogger", null);
+
+    b1 = new MDAOPutBanchmark();
+
+    r1 = new BenchmarkRunner.Builder(x).setInvocationCount(1000).setBenchmark(b1).build();
+    r1.execute(x);""",
+      "lastModifiedBy": 1348
+    })
+})
+
+
+p({
+  "class": "foam.nanos.script.Script",
+  "id": "benchmarkMDAOPutFind",
+  "description": "Benchmark MDAO put and find",
+  "code": """import foam.nanos.bench.*;
+    import foam.nanos.bench.benchmarks.*;
+    import net.nanopay.bench.*;
+
+    x = x.put("pmLogger", null);
+
+    b2 = new MDAOPutFind();
+
+    r2 = new BenchmarkRunner.Builder(x).setThreadCount(1).setInvocationCount(1000).setBenchmark(b2).build();
+    r2.execute(x);""",
+      "lastModifiedBy": 1348
```

Comment:
Why is this line indented?

Reply:
no reason other then copy-pasting

fixed